src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.php

```diff
@@ -11,26 +11,32 @@
 
 namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
+use Symfony\Component\HttpKernel\Debug\ErrorHandlerConfigurator;
 use Symfony\Component\HttpKernel\Debug\FileLinkFormatter;
 use Symfony\Component\HttpKernel\EventListener\DebugHandlersListener;
 
 return static function (ContainerConfigurator $container) {
     $container->parameters()->set('debug.error_handler.throw_at', -1);
 
     $container->services()
-        ->set('debug.debug_handlers_listener', DebugHandlersListener::class)
+        ->set('debug.error_handler_configurator', ErrorHandlerConfigurator::class)
+            ->public()
             ->args([
-                null, // Exception handler
                 service('monolog.logger.php')->nullOnInvalid(),
                 null, // Log levels map for enabled error levels
                 param('debug.error_handler.throw_at'),
                 param('kernel.debug'),
                 param('kernel.debug'),
                 service('monolog.logger.deprecation')->nullOnInvalid(),
             ])
-            ->tag('kernel.event_subscriber')
             ->tag('monolog.logger', ['channel' => 'php'])
 
+        ->set('debug.debug_handlers_listener', DebugHandlersListener::class)
+            ->args([
+                null, // Exception handler
```

Comment:
abstract arg?

Reply:
As far as I can tell, this is actually never set anywhere, it remains `null`. Maybe it'd be best to remove it?

Reply:
yes, let's remove it from this file (but keep the arg in the class just in case)

Reply:
Done